src/controllers/molecules/MessageComposer.js

```diff
@@ -228,6 +254,27 @@ module.exports = {
     onEnter: function(ev) {
         var contentText = this.refs.textarea.value;
 
+        // bodge for now to set markdown state on/off. We probably want a separate
+        // area for "local" commands which don't hit out to the server.
+        if (contentText.indexOf("/markdown") === 0) {
+            ev.preventDefault();
+            this.refs.textarea.value = '';
+            if (contentText.indexOf("/markdown on") === 0) {
+                this.markdownEnabled = true;
+            }
+            else if (contentText.indexOf("/markdown off") === 0) {
+                this.markdownEnabled = false;
+            }
```

Comment:
Is there a reason this comes with an on/off switch? It would only affect people if they typed /markdown presumably? Having to turn it on every time I refresh my browser is a major pain.